src/Map/Map.js

```diff
@@ -119,6 +122,8 @@ Map.propTypes = {
   fullScreen: PropTypes.bool,
   /** optionally pass a custom map */
   map: PropTypes.object,
+  /** place the ol-kit logo on the 'left', 'right', or set to 'none' to hide */
+  mapLogoPosition: PropTypes.string,
```

Comment:
Since we're adding this prop to the map I'd vote to shorten this to `logoPosition` since as a prop to map, I think it's pretty clear what it applies to